apps/condo/domains/banking/schema/BankAccount.js

```diff
@@ -44,11 +44,6 @@ const BankAccount = new GQLListSchema('BankAccount', {
                     }
                 },
             },
-            access: {
-                read: true,
-                create: true,
-                update: false,
```

Comment:
How will be handled a case when we're trying to change existing integration context?